examples/custom-metrics/README.md

```diff
@@ -1,113 +1,75 @@
 ---
-title: Example Flower App with Custom Metrics
 tags: [basic, vision, fds]
 dataset: [CIFAR-10]
 framework: [tensorflow]
 ---
 
-# Flower Example using Custom Metrics
+# Custom Metrics for Federated Learning with TensorFlow and Flower
 
 This simple example demonstrates how to calculate custom metrics over multiple clients beyond the traditional ones available in the ML frameworks. In this case, it demonstrates the use of ready-available `scikit-learn` metrics: accuracy, recall, precision, and f1-score.
 
-Once both the test values (`y_test`) and the predictions (`y_pred`) are available on the client side (`client.py`), other metrics or custom ones are possible to be calculated.
+Once both the test values (`y_test`) and the predictions (`y_pred`) are available on the client side (`client_app.py`), other metrics or custom ones are possible to be calculated.
 
 The main takeaways of this implementation are:
 
-- the use of the `output_dict` on the client side - inside `evaluate` method on `client.py`
-- the use of the `evaluate_metrics_aggregation_fn` - to aggregate the metrics on the server side, part of the `strategy` on `server.py`
+- the use of the `output_dict` on the client side - inside `evaluate` method on `client_app.py`
```

Comment:
```suggestion
- the return of metrics generated at the `evaluate` method on `client_app.py`
```

Reply:
I'm thinking mentioning the `output_dict` is so specific to this example that it might be better to phrase the takeaways in more general terms?

Reply:
Good point, I've reworded it to `the return of multiple evaluation metrics generated ...`